grpc/README.md

```diff
@@ -0,0 +1,39 @@
+# gRPC Application
+
+A basic gRPC microservice architecture, involving a gRPC server and a
+client, that allow users to buy items from an online store.
+
+The ecommerce interface provided by the gRPC server, allows users to
+list and buy grocery via the public client/server application.
+
+The client/server application is deployed as a Code Engine application,
+with a public endpoint. While the gRPC server is deployed as a Code Engine
+application only exposed to the Code Engine project.
+
+See image:
+
+
+## Source code
+
+Check the source code if you want to understand how this works. In general,
+we provided three directories:
+
+- `/ecommerce` directory hosts the protobuf files and declares the `grocery`
```

Comment:
Is there every a need to regenerate this directory (e.g. after user made some adjustments to play around)? If so, I suggest to add the command to re-generate the files.

Reply:
done. Added a new script call `./regenerate-grpc-code.sh